google/cloud/speech/cloud_speech.yaml

```diff
@@ -12,18 +12,8 @@ documentation:
 
 apis:
 - name: google.cloud.speech.v1beta1.Speech
-# TODO get LRO API call support added for Speech
-#- name: google.longrunning.Operations
+- name: google.longrunning.Operations
```

Comment:
So I don't understand what's different between IAM and LRO as far as this design goes. Are the IAM methods in Pub/Sub modified or given context in some manner via being a mixin? IOW, do they behave somehow differently than they would if they were rolled in a separate GAPIC package?